src/print.rs

```diff
@@ -36,7 +37,59 @@ pub fn prompt(args: ArgMatches) {
         writeln!(handle).unwrap();
     }
 
-    let modules = PROMPT_ORDER
+    let mut prompt_order: Vec<&str> = Vec::new();
+    //List of default prompt order
+    let default_prompt_order: &[&str] = &[
```

Comment:
I think we should be able to make `default_prompt_order` a `const` rather than a `let`, to avoid the variable assignment.